storage/mysql/log_admin.go

```diff
@@ -0,0 +1,78 @@
+package mysql
+
+import (
+	"fmt"
+
+	"github.com/google/trillian/crypto"
+	"github.com/google/trillian/merkle"
+	"github.com/google/trillian/storage/cache"
+)
+
+const (
+	setTreePropertiesSQL = `INSERT INTO Trees(TreeId,KeyId,TreeType,LeafHasherType,TreeHasherType,AllowsDuplicateLeaves) VALUES(?, ?, ?, ?, ?, ?)`
+	setTreeParametersSQL = `INSERT INTO TreeControl(TreeId,ReadOnlyRequests,SigningEnabled,SequencingEnabled,SequenceIntervalSeconds,SignIntervalSeconds) 
+		VALUES(?, ?, ?, ?, ?, ?)`
+	deleteTreeSQL1 = `DELETE FROM Trees WHERE TreeId = ?`
+	deleteTreeSQL2 = `DELETE FROM TreeControl WHERE TreeId = ?`
+)
+
+// CreateLog instantiates a new log with default parameters.
+// TODO(codinglama): Move to admin API when the adnmin API is created.
+func CreateLog(logID int64, dbURL string) error {
+	th := merkle.NewRFC6962TreeHasher(crypto.NewSHA256())
+	m, err := newTreeStorage(logID, dbURL, th.Size(), defaultLogStrata, cache.PopulateLogSubtreeNodes(th))
+	if err != nil {
+		return fmt.Errorf("Couldn't create a new treeStorage: %s", err)
+	}
+	// Set Tree Row
+	stmt, err := m.db.Prepare(setTreePropertiesSQL)
+	if err != nil {
+		return err
+	}
+	defer stmt.Close()
+	_, err = stmt.Exec(logID, 1, "MAP", "SHA256", "SHA256", false)
```

Comment:
nit: Could we have "1" extracted to a named variable?

Also, did you mean "LOG" instead of "MAP"? The method is misnamed otherwise. If you want a generic method, maybe rename to CreateTree() and pass the type as parameter?